src/FluentValidation.Tests/LanguageManagerTests.cs

```diff
@@ -101,6 +101,32 @@ public void Gets_translation_for_croatian_culture()
 		}
 
 
+		[Fact]

```

Comment:
Should probably make this a [Fact(Skip="Explicit")] for now as we know it's going to fail until the messages are re-translated.

Reply:
As of this PR they shouldn't fail though.  Are you seeing failures?
Sure, hard-coding 1000 isn't great but as it is it shows -1 which is surely worse...
I could hard-code an infinity symbol instead!
All those others issues with the translations were picked up by the test.
BTW there is a potential bug in my code if you happened to have a message that had 3 consecutive {'s - not sure if you're really worried about that possibility though.

Reply:
Ready to push a new commit if you've made a call on the use 1000 (I did even try replacing with ∞ : not sure how, e.g.  "'Code' doit contenir entre 2 et ∞ caractères" would read to a French speaker though!)

Reply:
@wizofaus Yes you were right, the test does pass (I'd modified something when I was testing).

Let's go with the 1000 characters